### ISSUE TITLE
Client does not notice when server is no longer responding

### ISSUE THREAD

**Issue by dietmarw** [seripap/darkwire.io#23]
We experienced that it happens from time to time that the server is "loosing" the room but the client still thinks it's connected and sends off message after message and even shows the other participants still present. Unfortunately I'm not sure how to best reproduce this properly.

**Comment by dietmarw** [comment]
Got some logs now. Seems a time out is reached when there is no activity for a certain time. In the last case it was something over an hour:
```
at=error code=H15 desc="Idle connection" method=GET path="/socket.io/?EIO=3&
transport=websocket&sid=je4XMm92GzQItCgIAAAC" host=moleaks.herokuapp.com 
request_id=6ca93f60-4ed1-489b-881f-124c3792915d fwd="XX.XX.XX.XX" dyno=web.1
connect=1ms service=1145043ms status=503 bytes=309
```

Might actually just be heroku related.

**Comment by seripap** [comment]
We haven't done extensive testing on chat idling but there does not to be a better way to indicate online versus offline. Something will come in the pipeline in the near future.

**Comment by dietmarw** [comment]
:+1:

**Comment by seripap** [comment]
Will be fixed in #26
